src/cpp/session/resources/schema/user-state-schema.json

```diff
@@ -4,6 +4,139 @@
     "title": "RStudio User State",
     "type": "object",
     "properties": {
+        "general": {
+            "type": "object",
+            "properties": {
+                "ignoredUpdateVersions": {
+                    "type": "array",
+                    "description": "A list of ignored updates. Ignored updates will not trigger an available update.",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "clipboardMonitoring": {
+                    "type": "boolean",
+                    "description": "Linux clipboard monitoring for middle-click paste"
+                }
+            },
+            "default": {
+                "ignoredUpdateVersions": [],
+                "clipboardMonitoring": true
+            }
+        },
+        "font": {
+            "type": "object",
+            "properties": {
+                "proportionalFont": {
+                    "type": "string"
+                },
+                "fixedWidthFont": {
+                    "type": "string"
+                },
+                "useFontConfigDb": {
```

Comment:
Do you know if we still need this?

Reply:
I'll take it out since the font DB isn't necessary now but I also wasn't sure what performance would be like when you mentioned that some users have a large number of fonts installed. I suppose the font DB wouldn't be the first choice when dealing with that performance. If there were a performance problem with a lot of fonts, I would probably refactor the call to retrieve the fonts asynchronously and caching it instead of building the cache at startup.